Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -78,17 +75,53 @@ - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVide
 
 - (BOOL)answerCall:(OCTCall *)call enableAudio:(BOOL)enableAudio enableVideo:(BOOL)enableVideo error:(NSError **)error
 {
-    return NO;
+    OCTToxAVAudioBitRate audioBitRate = (enableAudio) ? kDefaultAudioBitRate : kOCTToxAVAudioBitRateDisable;
+    OCTToxAVVideoBitRate videoBitRate = (enableVideo) ? kDefaultVideoBitRate : kOCTToxAVVideoBitRateDisable;
+
+    if (call.chat.friends.count == 1) {
+
+        OCTFriend *friend = call.chat.friends.firstObject;
+
+        BOOL status = [self.toxAV answerIncomingCallFromFriend:friend.friendNumber audioBitRate:audioBitRate videoBitRate:videoBitRate error:error];
+
+        [self.calls updateCallWithChat:call.chat updateBlock:^(OCTCall *call) {
+            call.status = (status) ? OCTCallStatusActive : OCTCallStatusInactive;
```

Comment:
As documentation says
>  If answering fails for any reason, the call will still be pending and it is possible to try and answer it later.

So in case of error you don't want to change status to Inactive.